src/send/SendConfirmation.tsx

```diff
@@ -1,74 +1,90 @@
 import { sendPayment } from 'src/send/actions'
 import { isSendingSelector } from 'src/send/selectors'
 import { usePrepareSendTransactions } from 'src/send/usePrepareSendTransactions'
-import DisconnectBanner from 'src/shared/DisconnectBanner'
-import colors from 'src/styles/colors'
-import { typeScale } from 'src/styles/fonts'
+import { NETWORK_NAMES } from 'src/shared/conts'
 import { useAmountAsUsd, useTokenInfo, useTokenToLocalAmount } from 'src/tokens/hooks'
 import { feeCurrenciesSelector } from 'src/tokens/selectors'
+import Logger from 'src/utils/Logger'
 import { getFeeCurrencyAndAmounts } from 'src/viem/prepareTransactions'
 import { getSerializablePreparedTransaction } from 'src/viem/preparedTransactionSerialization'
 import { walletAddressSelector } from 'src/web3/selectors'
 
-type OwnProps = NativeStackScreenProps<
+type Props = NativeStackScreenProps<
   StackParamList,
   Screens.SendConfirmation | Screens.SendConfirmationModal
 >
-type Props = OwnProps
 
 const DEBOUNCE_TIME_MS = 250
+const TAG = 'send/SendConfirmation'
 
 export const sendConfirmationScreenNavOptions = noHeader
 
-function SendConfirmation(props: Props) {
+export default function SendConfirmation(props: Props) {
   const { t } = useTranslation()
+  const dispatch = useDispatch()
 
   const {
     origin,
     transactionData: { recipient, tokenAmount, tokenAddress, tokenId },
   } = props.route.params
 
-  const { prepareTransactionsResult, refreshPreparedTransactions, clearPreparedTransactions } =
-    usePrepareSendTransactions()
-
-  const { maxFeeAmount, feeCurrency: feeTokenInfo } =
-    getFeeCurrencyAndAmounts(prepareTransactionsResult)
+  const {
+    prepareTransactionsResult,
+    refreshPreparedTransactions,
+    clearPreparedTransactions,
+    prepareTransactionLoading,
+  } = usePrepareSendTransactions()
 
   const tokenInfo = useTokenInfo(tokenId)
   const isSending = useSelector(isSendingSelector)
-  const fromModal = props.route.name === Screens.SendConfirmationModal
   const localCurrencyCode = useSelector(getLocalCurrencyCode)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol) ?? LocalCurrencySymbol.USD
   const localAmount = useTokenToLocalAmount(tokenAmount, tokenId)
   const usdAmount = useAmountAsUsd(tokenAmount, tokenId)
-
   const walletAddress = useSelector(walletAddressSelector)
   const feeCurrencies = useSelector((state) => feeCurrenciesSelector(state, tokenInfo!.networkId))
-
-  const dispatch = useDispatch()
+  const { maxFeeAmount, feeCurrency: feeTokenInfo } =
+    getFeeCurrencyAndAmounts(prepareTransactionsResult)
+  const tokenFeeAmount = maxFeeAmount ?? new BigNumber(0)
+  const localFeeAmount = useTokenToLocalAmount(tokenFeeAmount, feeTokenInfo?.tokenId)
+
+  const networkFeeDisplayAmount = useMemo(
+    () => ({
+      token: formatValueToDisplay(tokenFeeAmount),
+      local: localFeeAmount ? formatValueToDisplay(localFeeAmount) : undefined,
+    }),
+    [localFeeAmount, tokenFeeAmount, feeTokenInfo]
```

Comment:
```suggestion
    [localFeeAmount, tokenFeeAmount]
```
i wonder if this variable is necessary given that we only use it in one place below?

Reply:
Forgot to remove after the last changes to this variable. Will remove!